node.go

```diff
@@ -141,3 +142,49 @@ func (n *Node) References(refs *ua.NodeID) (*ua.BrowseResponse, error) {
 	return n.c.Browse(req)
 	// implement browse_next
 }
+
+//translate an array of browseName segment to NodeID
+func (n *Node) TranslateBrowsePathInMultiNamespaceToNodeId(pathNames []*ua.QualifiedName) (nodeId *ua.NodeID, err error) {
+	req := ua.TranslateBrowsePathsToNodeIDsRequest{
+		RequestHeader: &ua.RequestHeader{AuthenticationToken: ua.NewFourByteNodeID(0, id.TranslateBrowsePathsToNodeIDsRequest_Encoding_DefaultBinary)},
+		BrowsePaths: []*ua.BrowsePath{
+			{StartingNode: n.ID, RelativePath: &ua.RelativePath{Elements: []*ua.RelativePathElement{}}},
+		}}
+
+	for _, pathSegment := range pathNames {
+		req.BrowsePaths[0].RelativePath.Elements = append(req.BrowsePaths[0].RelativePath.Elements,
+			&ua.RelativePathElement{ReferenceTypeID: ua.NewTwoByteNodeID(id.HierarchicalReferences),
+				IsInverse:       false,
+				IncludeSubtypes: true,
+				TargetName:      pathSegment},
+		)
+	}
+
+	err = n.c.Send(&req, func(i interface{}) error {
+		if resp, ok := i.(*ua.TranslateBrowsePathsToNodeIDsResponse); ok {
+			if len(resp.Results) > 0 {
+				if resp.Results[0].StatusCode == ua.StatusOK {
+					if len(resp.Results[0].Targets) > 0 {
+						nodeId = resp.Results[0].Targets[0].TargetID.NodeID
+						return nil
+					}
+				} else {
+					return resp.Results[0].StatusCode
+				}
+			}
+		}
+		return ua.StatusBadUnexpectedError
+	})
+	return
+}
+
+//translate a browseName relative path to NodeID
```

Comment:
Is `browseNamePath` something that is used in OPC/UA or is this just the value of the `BrowseName` field? If yes, then just call it `browseName`. 

* NodeId -> NodeID
* don't name return values in the func declaration, e.g. `func (n *Node) Translate(...) (*ua.NodeID, error) { ... }`
* Also, try to keep the variable names short if the function is short, e.g.

```go
func ... {
    var names []*ua.QualifiedName
    segments := strings.Split(browseName, ".")
    for _, s := range segments {
        qn := &ua.QualifiedName{NamespaceIndex: uint16(ns), Name: s}
        names = append(names, qn)
    }
    return ...
}
```